spec/models/list_set_spec.rb

```diff
@@ -36,50 +37,51 @@
         ],
         page_size: SearchApiSearch::PAGE_SIZE_TO_GET_EVERYTHING,
       )
-
-      @list_set = ListSet.new("specialist_sector", "paye-content-id", @group_data)
     end
 
     it "returns the groups in the curated order" do
-      assert_equal ["Paying HMRC", "Annual PAYE and payroll tasks"], @list_set.map(&:title)
+      expect(list_set.map(&:title)).to eq(["Paying HMRC", "Annual PAYE and payroll tasks"])
     end
 
     it "provides the title and base_path for group items" do
-      groups = @list_set.to_a
+      groups = list_set.to_a
 
-      assert_equal "Employee tax codes", groups[1].contents.to_a[2].title
-      assert_equal "/pay-psa", groups[0].contents.to_a[1].base_path
+      expect(groups.[](1).contents.to_a.[](2).title).to eq("Employee tax codes")
```

Comment:
the array syntax here looks a little weird - it's converted `groups[1]` to `groups.[](1)` which looks like it's doing something clever when it isn't. (It's in a number of places)